onnxmltools/convert/coreml/operator_converters/neural_network/Activation.py

```diff
@@ -49,10 +52,6 @@ def convert_activation(scope, operator, container):
             op_type = 'Softsign'
         elif activation_type == 'softplus':
```

Comment:
Note that softplus is a different operation here! :)

Reply:
Yep, we can replace it if that op gets removed.